src/plays/typing-speed-test/components/ResultModal.js

```diff
@@ -0,0 +1,36 @@
+import { Modal } from "@mui/material";
+import "../styles.css";
+
+const ResultModal = ({ open, handleModalClose, wpm, cpm, accuracy }) => {

```

Comment:
```suggestion
const ResultModal = ({ open, onModalClose, stats }) => {

```
Here we can pass the one stats object which will contain this information.

```js
const stats = { wpm, cpm, accuracy }
```